samza-rest/src/main/java/org/apache/samza/monitor/LocalStoreMonitor.java

```diff
@@ -83,9 +83,10 @@ public void monitor() throws Exception {
                              String.format("%s-%s", jobInstance.getJobName(), jobInstance.getJobId()));
       try {
         JobStatus jobStatus = jobsClient.getJobStatus(jobInstance);
+        LOG.info("Job: {} has the status: {}.", jobInstance, jobStatus);
         for (Task task : jobsClient.getTasks(jobInstance)) {
+          LOG.info("  Evaluating stores for job: ({}) task: ({})", jobInstance, task);
```

Comment:
Yes, whitespace is for readability. 

Parens were there to help distinguish the job string representation from the task string representation. Removed the job, since it should be obvious with the indentation.

Looks like this:

> 2017-06-14 21:50:48.154 [pool-1-thread-1] LocalStoreMonitor [INFO] Job: jobName:my-job-name jobId:i001 has the status: started.
2017-06-14 21:50:48.154 [pool-1-thread-1] LocalStoreMonitor [INFO]   Evaluating stores for task: taskName:Partition 288 container:1 preferredHost:some.host.com stores:[store1, store2, store3]
2017-06-14 21:50:48.154 [pool-1-thread-1] LocalStoreMonitor [INFO]   Local store: store1 not used by the task: Partition 288.